API/DTOs/PlayerMatchStatsDTO.cs

```diff
@@ -1,20 +1,71 @@
 namespace API.DTOs;
 
 /// <summary>
-/// Used for POSTing match statistics to the API
+/// Represents a player's match stats
 /// </summary>
 public class PlayerMatchStatsDTO
 {
-    public int Id { get; set; }
+    /// <summary>
+    /// The id of the player
+    /// </summary>
+    public int PlayerId { get; set; }
+    /// <summary>
+    /// The id of the match
+    /// </summary>
     public int MatchId { get; set; }
```

Comment:
Add whitespace in between properties :) I wonder if there is an editorconfig setting for that

Reply:
Also suppress any warnings about unused properties here and in the entity class